Engine/DataFeeds/FileSystemDataFeed.cs

```diff
@@ -40,18 +39,18 @@ namespace QuantConnect.Lean.Engine.DataFeeds
     /// <remarks>Filesystem datafeeds are incredibly fast</remarks>
     public class FileSystemDataFeed : IDataFeed
     {
-        /******************************************************** 
+        /********************************************************
         * CLASS VARIABLES
         *********************************************************/
         // Set types in public area to speed up:
-        private IAlgorithm _algorithm;
-        private BacktestNodePacket _job;
-        private bool _endOfStreams = false;
-        private int _subscriptions = 0;
-        private int _bridgeMax = 500000;
-        private bool _exitTriggered = false;
-
-        /******************************************************** 
+        private readonly IAlgorithm _algorithm;
+        private readonly BacktestNodePacket _job;
+        private bool _endOfStreams;
+        private readonly int _subscriptions;
```

Comment:
We'll soon add universe selection where the feed count changes - the subscriptions will need to be dynamic. This change can stay for now though until we start universe implementation